src/pages/dmt5/index.tsx

```diff
@@ -59,6 +61,32 @@ const DMT5 = () => {
         window.addEventListener('resize', handleResizeWindow)
     }, [])
 
+    const target = '#gatsby-focus-wrapper'
```

Comment:
Could u pls move `target`, `options` and `lazy_components` out of the DMT5 :pray: